### DISCUSSION TITLE
Welcome to ChatPaper Discussions!

### DISCUSSION THREAD

**Discussion by kaixindelele** [kaixindelele/ChatPaper#55] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by HeathBrothers** [comment]
there is an error when i ran:python chat_paper.py --pdf_path "demo.pdf", 
error: ModuleNotFoundError: No module named 'arxiv'
but actually I have install the module "arxiv"
Requirement already satisfied: arxiv in /Users/jingxiao/opt/anaconda3/lib/python3.9/site-packages (1.4.3)
Requirement already satisfied: feedparser in /Users/jingxiao/opt/anaconda3/lib/python3.9/site-packages (from arxiv) (6.0.10)
Requirement already satisfied: sgmllib3k in /Users/jingxiao/opt/anaconda3/lib/python3.9/site-packages (from feedparser->arxiv) (1.0.0)

run enviroment: /Users/jingxiao/opt/anaconda3/lib/python3.9

**Comment by kaixindelele** [comment]
都已经用anaconda了，你建个虚拟环境吧

**Comment by HeathBrothers** [comment]
我建立虚拟环境后，仍然报错。
pip install -r requirements.txt之后

(myenv-chatpaper) ***@***.*** ChatPaper-main % python chat_paper.py --pdf_path "demo.pdf"
Traceback (most recent call last):
 File "chat_paper.py", line 5, in
   import arxiv
ModuleNotFoundError: No module named 'arxiv'




| |
man_in_the_mirror
|
|
***@***.***
|


---- Replied Message ----
| From | ***@***.***> |
| Date | 03/21/2023 20:40 |
| To | ***@***.***> |
| Cc | ***@***.***> ,
***@***.***> |
| Subject | Re: [kaixindelele/ChatPaper] Welcome to ChatPaper Discussions! (Discussion #55) |

都已经用anaconda了，你建个虚拟环境吧
